examples/mail/archiveItems.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+require_once "vendor/autoload.php";
+
+use garethp\ews\API\Enumeration\DistinguishedFolderIdNameType;
+use garethp\ews\API\ExchangeWebServices;
+use garethp\ews\API\Type;
+use garethp\ews\API\Type\DistinguishedFolderIdType;
+
+$ews = ExchangeWebServices::withUsernameAndPassword('server', 'username', 'password');
+
+$request = [
+    'ArchiveSourceFolderId' => (new DistinguishedFolderIdType(DistinguishedFolderIdNameType::INBOX))->toArray(true),
+    'ItemIds' => [
+        'ItemId' => array_map(function($itemId) {
```

Comment:
This might be a touch clearer if you just had an array of the two items, rather than passing them through `array_map`. This will also clear up the code style error that's being thrown by phpcs

Reply:
Sure, made that change.